mmv1/products/compute/ResizeRequest.yaml

```diff
@@ -0,0 +1,168 @@
+
+### Adjust when ready for GA ###
+min_version: beta
+immutable: true
+
+### List Method ###
+base_url: "projects/{{project}}/zones/{{zone}}/instanceGroupManagers/{{instanceGroupManager}}/resizeRequests"
+
+### Get Method
+self_link: "projects/{{project}}/zones/{{zone}}/instanceGroupManagers/{{instanceGroupManager}}/resizeRequests/{{resizeRequest}}"
+
+### Create Method ###
+# Get method with a post
+create_verb: :POST
+
+### Update method ###
+# Resize requests are currently not update-able
+
+### Delete Method ###
+# Custom delete method to handle resize request cancellations vs. deletions.
+# If a resize request is in the ACCEPTED state, it must be canceled before it can be
+# deleted. If a resize request is NOT in the ACCEPTED state, it can be directly deleted.
+custom_code: !ruby/object:Provider::Terraform::CustomCode
+  custom_delete: templates/terraform/custom_delete/compute_mig_resize_request_delete.go.erb
+
+# If true, code for handling long-running operations is generated along with
+# the resource. If false, that code is not generated.
+autogen_async: true
+# Sets parameters for handling operations returned by the API.
+async: !ruby/object:Api::OpAsync
+  # Overrides which API calls return operations. Default: ['create',
+  # 'update', 'delete']
+  # actions: ['create', 'update', 'delete']
+  operation: !ruby/object:Api::OpAsync::Operation
+    kind: "compute#operation"
+    path: "name"
+    base_url: "projects/{{project}}/zones/{{zone}}/operations/{{op_id}}"
+    wait_ms: 1000
+  result: !ruby/object:Api::OpAsync::Result
+    path: "targetLink"
+  status: !ruby/object:Api::OpAsync::Status
+    path: "status"
+    complete: "DONE"
+    allowed:
+      - "PENDING"
+      - "RUNNING"
+      - "DONE"
+  error: !ruby/object:Api::OpAsync::Error
+    path: "error/errors"
+    message: "message"
+
+# Resize request parameters injected via URL
+parameters:
+  - !ruby/object:Api::Type::ResourceRef
+    name: "zone"
+    resource: "Zone"
+    imports: "name"
+    description: |
+      Name of the compute zone scoping this request. Name should conform to RFC1035.
+    required: true
+    url_param_only: true
+  - !ruby/object:Api::Type::ResourceRef
+    name: "instanceGroupManager"
+    resource: "InstanceGroupManager"
+    imports: "name"
+    description: |
+      The name of the managed instance group. The name should conform to RFC1035 or be a resource ID.
+
+      Authorization requires the following IAM permission on the specified resource instanceGroupManager:
+      *compute.instanceGroupManagers.update
+    required: true
+    url_param_only: true
+  - !ruby/object:Api::Type::ResourceRef
+    name: "resizeRequest"
+    resource: "ResizeRequest"
+    imports: "name"
+    description: |
+      The name of an existing resize request. The name should conform to RFC1035 or be a resource ID.
+    url_param_only: true
+
+# Resize request parameters sent in the body of the CRUD action
```

Comment:
seeing there is creation timestamp I would consider phrasing this differently (cause we don't send the output params) - unless this is the same for other similar requests.
"Resize request parameters"?